roles/openshift_health_checker/openshift_checks/efk_stack_check.py

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/env python
+'''
+  Module for performing checks on an elasticseatch, fluentd, and kibana stack
+'''
+# pylint: disable=missing-docstring
+# vim: expandtab:tabstop=4:shiftwidth=4
+#
+#   Copyright 2017 Red Hat Inc.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+import json
+import os
+import ssl
+import urllib2
+
+from openshift_checks import OpenShiftCheck, get_var
+
+
+class EFKStackCheck(OpenShiftCheck):
+    """
+        This is a check for the entire EFK stack shipped with OCP
+        https://github.com/openshift/openshift-tools/blob/prod/scripts/monitoring/cron-send-logging-checks.py
```

Comment:
The indentation here is off, and our CI doesn't check that.
Dedent one level please.

Also how relevant is it to link to that script? Isn't it all here self-contained?
The comment is not clear as to what is the point of the link.